src/main/java/net/hypixel/nerdbot/command/UserCommands.java

```diff
@@ -0,0 +1,150 @@
+package net.hypixel.nerdbot.command;
+
+import com.freya02.botcommands.api.application.ApplicationCommand;
+import com.freya02.botcommands.api.application.annotations.AppOption;
+import com.freya02.botcommands.api.application.slash.GuildSlashEvent;
+import com.freya02.botcommands.api.application.slash.annotations.JDASlashCommand;
+import com.joestelmach.natty.DateGroup;
+import com.joestelmach.natty.Parser;
+import com.mongodb.client.model.Filters;
+import com.mongodb.client.result.InsertOneResult;
+import lombok.extern.log4j.Log4j2;
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.hypixel.nerdbot.NerdBotApp;
+import net.hypixel.nerdbot.api.database.model.reminder.Reminder;
+import net.hypixel.nerdbot.util.discord.DiscordTimestamp;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+@Log4j2
+public class UserCommands extends ApplicationCommand {
+
+    @JDASlashCommand(name = "remind", subcommand = "create", description = "Set a reminder")
+    public void createReminder(GuildSlashEvent event, @AppOption String time, @AppOption String description) {
+        Date date;
+
+        try {
+            Parser parser = new Parser();
+            List<DateGroup> groups = parser.parse(time);
+            DateGroup group = groups.get(0);
+
+            List<Date> dates = group.getDates();
+            date = dates.get(0);
+        } catch (IndexOutOfBoundsException exception) {
+            try {
+                date = parseCustomFormat(time);
```

Comment:
do you explain to the user that they are able to use custom formats? how do they know this is a feature

Reply:
I do now